.github/workflows/ci-generated.yml

```diff
@@ -0,0 +1,39 @@
+name: Generate Files
+
+on:
+  push:
+      branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    # Check out master, so that we can count.
+    - uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: "3.7"
+    - name: Generate Files
+      # Record the count in a tmp file so it survives changing branches
+      run: |
+        make init-ci && make all
+        sed -i.bak '/*_generated.rs/d' .gitignore
+    - name: Commit files
+      # fails if no files to commit
+      continue-on-error: true
+      run: |
+        git add .
```

Comment:
this will add `.gitignore.bak` as well
I guess `.gitignore.bak` can be removed in the previous step.